### PR TITLE
Dockerized HTTP server

### PR DESCRIPTION
**Pull request by privateOmega**
@MatthewVita PTAL at my dockerize approach, I hope this should suffice.

### CODE REVIEW

**Comment by MatthewVita** [comment]
@privateOmega Hi. I added my bits. Please let me know if it looks good.

**Comment by MatthewVita** [comment]
I won't have time to look into this immediately, but I wanted to point out that there may be a bug somewhere. See the "UNKNOWN"s. Just posting this more as a reminder for myself. Feel free to ignore.

issue

**Comment by privateOmega** [comment]
> @privateOmega Hi. I added my bits. Please let me know if it looks good.

Thanks for filling in the README, I completely forgot about that.

File: docker/server.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 const nodeHL7Instance = new NodeHL7Complete()
 
-// TODO: This is crazy! So many API changes. I'm using 'body' for
-//       everything (for now). May be best to look for a totally 
-//       new middleware for this...
-const bodyParser = require('body-parser')
-app.use(bodyParser.text());
+// Latest Express middleware approach
+app.use(express.json())
+app.use(express.text())
```
Comment (privateOmega) [inline docker/server.js:9]: only concern of mine in this PR is, why do we have 2 middlewares for body parsing, wouldn't `express.json()` do?

**Comment by privateOmega** [comment]
All looks good, I am merging it into your branch.

**Comment by privateOmega** [comment]
@MatthewVita I've crossposted it as #24

**Comment by MatthewVita** [comment]
@privateOmega thanks for reviewing. Okay to merge (and maybe bump)?

I saw the crosspost. I won't have time to look at this for a few days. Obviously very important and the tests are clearly not serving us.

**Comment by privateOmega** [comment]
> @privateOmega thanks for reviewing. Okay to merge (and maybe bump)?
> 
> I saw the crosspost. I won't have time to look at this for a few days. Obviously very important and the tests are clearly not serving us.

Yup I guess it's good to merge. Bump we have to do it considering npm won't allow deploying same version again,  maybe patch update only considering it doesn't modify any functionality and is just addon, and some doc changes.

**Pull request merged**